tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaSSLChannelTest.java

```diff
@@ -113,6 +114,15 @@ public static Object[] instances() {
         };
     }
 
+    @DataProvider(name = "ports")
```

Comment:
KafkaSSLChannelTest is a class that is also in KOP
usually we keep those files mostly untouched and we create another sibling file like KafkaSSLChannelProxyTest that inherits from the original test fail but it starts the proxy and changes the port to connect to

Reply:
also this test is not about the code you are you touching,
this is about client -> broker communications, and this PR is for proxy->broker and broker->broker

Reply:
in order to exercise your changes we have to set
`conf.setTlsHostnameVerificationEnabled(true);`

I have updated the test

Reply:
this is enough to exercise broker->broker communications in case of TLS
for the proxy we must change other tests

Reply:
Makes sense, I wasn't sure where to add the test.